uiowa/uiowa
# ITS Updates Sprint 228

# How to test

```
git checkout main && git pull && ddev blt ds --site=its.uiowa.edu
```
- Open https://its.uiowa.ddev.site/alerts in an anonymous browser window.
- Login and create an alert (outage or service degradation) and go straight to published.
- Refresh the anonymous alerts page. No change.
- Sync this branch down and repeat the steps above. The anonymous alerts page should update and show the open alert.

```
git checkout its_updates_sprint_228 && git pull
```

- Masquerade as an editor.
  - Note that you are unable to see the Fragments overview and that you can't see the Region Settings under /admin/config.
  - Note that you can't edit main menu items under Structure (probably can't see that menu item anyway).
  - Note that you can't edit the footer contact block.
- Masquerade as a user with both the editor and publisher roles
  - Note that you also can't see the Fragments overview...
  - Note that you also can't see the main menu settings and when editing a page you can't see them there.
  - Note that you can't edit the footer contact block but that you are able to add/edit/delete a text area within page's layout builder.
